components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java

```diff
@@ -367,14 +389,118 @@ public APIDefinitionValidationResponse validateAPIDefinition(String apiDefinitio
         return validationResponse;
     }
 
+    /**
+     * Populate definition with wso2 APIM specific information
+     *
+     * @param oasDefinition OAS definition
+     * @param swaggerData   API
+     * @return Generated OAS definition
+     * @throws APIManagementException If an error occurred
+     */
+    @Override
+    public String populateCustomManagementInfo(String oasDefinition, SwaggerData swaggerData)
+            throws APIManagementException {
+        Swagger swagger = getSwagger(oasDefinition);
+        Map<String, Object> extensions = swagger.getVendorExtensions();
```

Comment:
need to do a null check here. this is causing failure in the sample Pizza shack api creation